lib/cc/yaml/nodes/root.rb

```diff
@@ -16,6 +20,17 @@ def nested_warnings(*)
         def inspect
           "#<#{self.class.name}:#{super}>"
         end
+
+        def exclude?(path)
+          exclude_paths.present? &&
+            exclude_paths.match_any?(path)
+        end
+
+        def rate?(path)
+          ratings.nil? ||
+            ratings.paths.blank? ||
```

Comment:
Also, as per the commit message: not specifying means "rate all", is that what we want?